assets/i18n/hi.json

```diff
@@ -1,301 +1,302 @@
-	"I18N_SIGNUP_UPDATE_WARNING": "कृपया ध्यान दें कि हमने हाल ही में हमारी उपयोग की शर्तों को नवीनीकृत किया है ।",
-	"I18N_SIGNUP_USERNAME": "उपयोगकर्ता नाम",
-	"I18N_SIGNUP_USERNAME_EXPLANATION": "आपका उपयोगकर्ता नाम आपके योगदान के बगल में दिखाया जाएगा।",
-	"I18N_SIGNUP_WAIVER_OBJECTIVE": "वेवर ऑफ़ थे अत्त्रिबुतिओं  (BY) रेक्विरेमेंट अर्थात, यदि कोई इस काम को दोबारा इस्तेमाल करता है , उन्हें लेखकों को विशेषता देने की कोई आवस्यकता नहीं हैं। तथापि, आपके व्यक्तिगत योगदान के सभी विवरण साइट के एक्सप्लोरेशन लोग पर उपलब्ध रहेगी । जो लोग एक्सप्लोरेशन का पुनः प्रयोग करते है उन्हें हम प्रोत्साहित (आवश्यक नहीं) करते है की वे अपने काम में इस पेज की एक लिंक जोड़े ।",
-	"I18N_SIGNUP_WHY_LICENSE": "क्यों CC-BY-SA?",
-	"I18N_SPLASH_FIRST_EXPLORATION_DESCRIPTION": "Oppia के पाठ, जिन्हें हम एक्सप्लोरेशन्स भी कहते है , एक स्थिर वीडियो या पाठ की तुलना में अधिक गहरा अनुभव प्रदान करते है और सीखने में मदद करते है ।",
-	"I18N_SPLASH_JAVASCRIPT_ERROR_DESCRIPTION": "Oppia एक मुफ्त अथवा  ओपन सोर्स मंच है जो इंटरैक्टिव गतिविधियों से भरा है जिसे हम एक्सप्लोरेशन्स कहते है। अफसोस की बात है, Oppia को ठीक ढंग से काम करने के लिए आपके ब्राउज़र पे जावास्क्रिप्ट की जरुरत है  परंतु आपके ब्राउज़र पे जावास्क्रिप्ट उपलब्ध नहीं है। अगर आपको जावास्क्रिप्ट हेतु मदद की जरुरत है, <a href=\"<[hrefUrl]>\">यहां क्लिक करे</a>",
-	"I18N_SPLASH_JAVASCRIPT_ERROR_THANKS": "धन्यवाद।",
-	"I18N_SPLASH_JAVASCRIPT_ERROR_TITLE": "हमें आपके ब्राउज़र के जावास्क्रिप्ट की आवश्यकता है।",
-	"I18N_SPLASH_PAGE_TITLE": "Oppia: सिखाओ, जानो और एक्स्प्लोर करो।",
-	"I18N_SPLASH_SECOND_EXPLORATION_DESCRIPTION": "एक्सप्लोरेशन्स बनाना काफी आसान है। वे आसानी से व्यक्तिगत छात्र प्रतिक्रिया और दुनिया भर में अपने शिक्षार्थियों के अनुभव के अनुसार अनुकूलित हो जाती है।",
-	"I18N_SPLASH_SITE_FEEDBACK": "साइट फीडबैक",
-	"I18N_SPLASH_SUBTITLE": "Oppia के द्वारा आसानी से इंटरैक्टिव पाठ बनाया जा सकता है जो सिखाता है अथवा संलग्न करता है।",
-	"I18N_SPLASH_THIRD_EXPLORATION_DESCRIPTION": "Oppia के द्वारा कई विषयो पर एक्सप्लोरेशन्स <a href=\"/dashboard?mode=create\">बनाया</a> और <a href=\"/library\">शेयर</a> किये जा सकते है , जो केवल आपकी कप्लना द्वारा सिमित है।",
-	"I18N_SPLASH_TITLE": "पुस्तकों के बाहर सोचे।",
-	"I18N_SUBSCRIBE_BUTTON_TEXT": "सदस्यता लें",
-	"I18N_TOPNAV_ABOUT": "जानिए",
-	"I18N_TOPNAV_ABOUT_OPPIA": "Oppia के बारे में",
-	"I18N_TOPNAV_ADMIN_PAGE": "व्यवस्थापक पेज",
-	"I18N_TOPNAV_BLOG": "ब्लॉग",
-	"I18N_TOPNAV_CONTACT_US": "संपर्क करें",
-	"I18N_TOPNAV_DASHBOARD": "क्रिएटर डैशबोर्ड",
-	"I18N_TOPNAV_DONATE": "दान करे",
-	"I18N_TOPNAV_FORUM": "मंच",
-	"I18N_TOPNAV_GET_STARTED": "शुरू हो जाओ",
-	"I18N_TOPNAV_LIBRARY": "पुस्तकालय",
-	"I18N_TOPNAV_LOGOUT": "लोग आउट करे",
-	"I18N_TOPNAV_MODERATOR_PAGE": "परिनियामक पेज",
-	"I18N_TOPNAV_NOTIFICATIONS": "सूचनाएं",
-	"I18N_TOPNAV_PREFERENCES": "प्राथमिकताएं",
-	"I18N_TOPNAV_SIGN_IN": "खाता खोले",
-	"I18N_TOPNAV_TEACH_WITH_OPPIA": "Oppia के साथ  सिखाये",
-	"I18N_UNSUBSCRIBE_BUTTON_TEXT": "सदस्यता समाप्त"
+    "I18N_ABOUT_PAGE_TITLE": "हमारे बारे में - Oppia",
```

Comment:
Can you leave tabs here, so it's easier to see the diff? I know it's not aligned with the other files in the codebase, but this is a quirk of translatewiki. Ditto for qqq.json.